packages/evm/contracts/voting/CRVoting.sol

```diff
@@ -301,6 +452,49 @@ contract CRVoting is Controlled, ICRVoting {
         return _vote.maxAllowedOutcome != OUTCOME_MISSING;
     }
 
+    /**
+    * @dev Tell if a representative currently represents another voter
+    * @param _voter Address of the principal being queried
+    * @param _representative Address of the representative being queried
+    * @return True if the representative currently represents the voter
+    */
+    function _isRepresentativeOf(address _voter, address _representative) internal view returns (bool) {
+        return representatives[_voter][_representative] || _voter == _representative;
+    }
+
+    /**
+    * @dev Verify if a representative address has been authorized by a voter for a specific vote
+    * @param _voteId Identification number of the vote to be checked
```

Comment:
```suggestion
    * @param _voteId ID of the vote instance to be checked
```

Reply:
Fixed in ff4b971e5307ec492cf063eeaa94c4e32ae9ad4f